cnd/src/config/file.rs

```diff
@@ -66,7 +64,7 @@ pub struct Logging {
 
 #[derive(Clone, Debug, Deserialize, Serialize, PartialEq)]
 pub struct Comit {
-    pub secret_seed: Seed,
+    pub secret_seed_file: PathBuf,
```

Comment:
I think you can make your life easier if we don't have this path.
A default path + the override from the commandline should be enough for now.
If anyone actually requests such a feature, we can always implement it. Doesn't seem to be worth the complexity for now :)

Reply:
Implemented as suggested.